cel/options.go

```diff
@@ -568,3 +568,13 @@ func ParserRecursionLimit(limit int) EnvOption {
 		return e, nil
 	}
 }
+
+// ParserExpressionSizeLimit adjusts the max. number of code points the expression parser is allowed
```

Comment:
nit: remove 'max.' and keep the first sentence on a single line.

Thanks!

Reply:
Done.